package/yast2-country.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Aug  3 16:38:40 UTC 2021 - José Iván López González <jlopez@suse.com>
+
+- Do not write an empty timezone (bsc#1188406).
```

Comment:
This is not helpful for the reader. I'll read the bug and suggest a better description